book/eha.qmd

```diff
@@ -81,17 +96,75 @@ $$\delta_{ie} = \II(Y_i \leq C_i \wedge E_i = e) \in \{0, 1\},\ e=1,\ldots,k$$
+<!-- Following @beyersmann.competing.2012, this book assumes that the cause-specific hazards model fully describes the data generating processes and doesn't require an independence assumption for the competing events. -->
 
 
 
 ## Multi-state Models {#sec-multi-state}
 
-{{< include _wip.qmd >}}
+Multi-state models can be considered the most general type of survival data, as other types (single-event, competing risks, recurrent events) can be viewed as special cases.
+
+A common multi-state model, the illness-death model, is illustrated in @fig-multi-state, left panel.
+The states are 'healthy' (State 0), with illness (State 1), or death (State 2).
+Healthy subjects can transition directly to death ($0\rightarrow 1$) or after previous illness ($0\rightarrow 1 \rightarrow 2$), without the possibility of back-transitions.
+Similar to competing risks, individual transitions are characterised by transition-specific hazards, denoted by $h_{\ell\rightarrow e}(\tau)$ or short $h_{\ell e}(\tau)$, where $\ell$ is the starting state and $e$ the end state  for a transition, $\ell, e \in \{0,\ldots,k\}$.
+Remaining in the same state (transition into the same sate $h_{\ell\ell}(\tau)$) is equivalent to being censored for any transition possible from that state.
+
+![Left: The illness-death model where subjects can transition from states: healthy (0), with illness (1), and terminal state death (2) without back-transitions. Right: Illustration of a general multi-state model with back-transitions. Here subjects can transition from state healthy (0) to different disease progressions (1, 2) with (partial) recovery (0, 1) and terminal state death (3).](Figures/survival/multi-state-examples-w-transitions.png){#fig-multi-state fig-alt="Schematic illustration of different multi-state models."}
+
+The right panel of @fig-multi-state depicts a more general multi-state setting with back-transitions.
+One can think of transitions between a healthy state (0) to different disease progressions (1, 2) with possibilty of improvement and recovery, and possible transition to a terminal state (3).
+
+When modeling such data, interest often lies in estimation of the transition hazards, but also in transition probabilities $P_{\ell e}(\zeta, \tau)$, that is the probability to transition from state $\ell$ to $e$ between time points $\zeta$ and $\tau$ or state occupation probabilities $P_e(\tau)$, the probability to occupy state $e$ at time $\tau$.
+Transition probabilities are often summarized in a matrix of transtion probabilities $\mathbf{P}(\zeta, \tau)$, where rows indicate starting states and columns indicate end states.
+
+For example, for the illness-death model, the transition matrix would be given as
+
+$$\mathbf{P}(\zeta, \tau) :=
+  \begin{pmatrix}
+  P_{00}(\zeta, \tau) & P_{01}(\zeta, \tau) & P_{02}(\zeta, \tau)\\
+  \bullet & P_{11}(\zeta, \tau) & P_{12}(\zeta, \tau)\\
+  \bullet & \bullet & P_{22}(\zeta, \tau)\\
+  \end{pmatrix}.
+$$
```

Comment:
Would it make sense to write either in the matrix or below that $P_{22}(\zeta, \tau) = 1$ and $P_{12}(\zeta, \tau) = 1 - P_{11}(\zeta, \tau)$ and $P_{02}(\zeta, \tau) = 1 - P_{00}(\zeta, \tau) - P_{01}(\zeta, \tau)$ ?